database/sql/create_functions.sql

```diff
@@ -26,29 +27,62 @@ END;
  * @param krome ID aktuální rozvrhové akce.
  * @return Místnost je volná.
  */
-CREATE OR REPLACE FUNCTION sem_je_mistnost_volna(p_mistnost_id NUMBER, p_den_v_tydnu NUMBER, p_od NUMBER, p_do NUMBER, krome NUMBER)
+CREATE OR REPLACE FUNCTION sem_je_mistnost_volna (p_mistnost_id NUMBER, p_den_v_tydnu NUMBER, p_od NUMBER, p_do NUMBER, krome NUMBER)
+    RETURN NUMBER AS
+        v_pocet NUMBER(1);
+  BEGIN
```

Comment:
Nebylo by lepší mít název tabulky (popř. další specifické údaje) v parametru, SQL složit z těchto parametrů a nemít tak více funkci, které mají podobnou logiku?

Reply:
Moc nevim co tim myslis, ale je mi jasny ze je ta moje funkce dost podobna te tvoji... Ale myslim ze to tak uz muzem nechat... Dali jsme tomu az moc hodin na to ze je to skolni projekt.